test/configure.js

```diff
@@ -0,0 +1,301 @@
+var assert = require( 'chai' ).assert;
+var fs = require( 'fs-extra' );
+var path = require( 'path' );
+
+var sugarcoat = require( '../index' );
+
+suite( 'Configure: Settings', () => {
+
+    test( 'Destination is set to be required. Destination errored out when not supplied.', (done) => {
+
+        var configMissingDest = {
+            sections: [
+                {
+                    title: 'CSS File',
+                    files: './test/assert/parseVarCode.css'
+                },
+                {
+                    title: 'CSS File 2',
+                    files: './test/assert/parseVarCode.css'
+                }
+            ]
+        };
+
+        sugarcoat( configMissingDest )
+        .then( data => {
```

Comment:
Need to assert in the resolve function as well as the reject. Say if Sugarcoat fails to reject when a dest is not provided, you need an assertion to say that this failed. It can be something simple!